### PR TITLE
[modules] Fix follow me

### PR DESCRIPTION
**Pull request by fvantienen**
This removes the possibilities of NaN's and adds the option to move multiples waypoints periodically with the follow_me module. An example flight plan will follow

### CODE REVIEW

File: sw/airborne/modules/ctrl/follow_me.c [hunk 1]
```diff
@@ -35,7 +36,7 @@
 
 // Height difference between te target be default 18 meters
 #ifndef FOLLOW_ME_HEIGHT
-#define FOLLOW_ME_HEIGHT 75
+#define FOLLOW_ME_HEIGHT 60
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/follow_me.c:39]: Some default values are not consistent with the comments

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]
just a few minor comments

**Review by dewagter — APPROVED** [review]
Important fix and nice new features. LGTM

File: sw/airborne/modules/ctrl/follow_me.c [hunk 2]
```diff
@@ -76,11 +76,72 @@ static struct LlaCoor_i ground_lla;
...
+    struct EnuCoor_f wp_new_enu;
+    wp_new_enu.x = wp_enu->x + diff_targetpos.x;
+    wp_new_enu.y = wp_enu->y + diff_targetpos.y;
+    wp_new_enu.z = wp_enu->z;
+
+    // Rotate the waypoint
+    float cos_heading = cosf(diff_targetpos_heading/180.*M_PI);
+    float sin_heading = sinf(diff_targetpos_heading/180.*M_PI);
+    wp_new_enu.x = ((wp_new_enu.x - cur_targetpos.x) * cos_heading) + ((wp_new_enu.y - cur_targetpos.y) * sin_heading) +  + cur_targetpos.x;
+    wp_new_enu.y = (-(wp_new_enu.y - cur_targetpos.y) * sin_heading) + ((wp_new_enu.y - cur_targetpos.y) * cos_heading) + cur_targetpos.y;
```
Comment (fvantienen) [inline sw/airborne/modules/ctrl/follow_me.c:134]: There is something wrong here with the x and y values

**Pull request merged**